lib/services.template.ejs

```diff
@@ -180,6 +180,12 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' &&
     (action.sharedMethod.aliases || []).forEach(function(alias) {
       var aliasMethod = alias.split('.').join('$');
 
+      // this is to handle strong-remoting does not prepend prototype.
+      // back into prototype methods
+      if (action.sharedMethod.isStatic === false) {
+        aliasMethod = ['prototype', aliasMethod].join('$');
```

Comment:
@bajtos previous convo conclusion is to just prepend prototype to avoid a breaking change in strong-remoting 

>**bajtos:** well, we can introduce a method to get a list of all methods & aliases
but e.g. the rest adapter does not want to de-duplicate aliases
**david:** right, ill take a look and fix this problem with patchAttributes, then maybe add an issue to see if we should explore " introduce a method to get a list of all methods & aliases"
**bajtos:** hmm, I just realised my proposal would be a breaking change in strong-remoting
because code expecting to see aliases without `prototype.` prefix will break after the change
so maybe it will be best to fix the problem in Angular SDK only for short-termuce a method to get a list of all methods & aliases" for longer term